.github/workflows/vg-publish-package-template.yml

```diff
@@ -0,0 +1,78 @@
+name: publish package to npm registry
+on: 
+  workflow_call:
+    secrets:
+      GT:  #created by default in workflow
+        required: true
+
+jobs:
+  build_and_test:
+    name: Build and test job
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo '::set-output name=dir::$(yarn cache dir)'
+
+      - uses: actions/cache@v2
+        name: validate cache
+        id: yarn-cache
+        with:
+          path: | 
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Installing yarn dependencies
+        run: yarn install --frozen-lockfile --prefer-offline
+
+      - name: Lint
+        run: yarn lint
+
+      #- run: npm i
+      - run: npm version prerelease --preid=ci-$GITHUB_RUN_ID --no-git-tag-version

```

Comment:
Good idea, could you use this one instead:
https://classic.yarnpkg.com/en/docs/cli/version
`yarn version` so that we have semantic versioning?